src/app/design/frontend/base/default/template/checkout/omise/success.phtml

```diff
@@ -14,70 +14,70 @@
 <p><?php echo $this->__( 'To complete payment, print the code below and bring it to 
 make payment in any Tesco Lotus Store. Code is valid for the next <b>24 hours only</b>. 
 After that time your order will be canceled.'); ?></p>
-<div class="omise omise-billpayment-tesco-details">
+<div class="omise omise-details">
```

Comment:
The file name here is `success.phtml` when the template is actually Tesco related